Tiltfile

```diff
@@ -165,6 +166,33 @@ for x in range(localnet_config["appgateservers"]["count"]):
         ],
     )
 
+# Provision Gateways
+actor_number = 0
+for x in range(localnet_config["gateways"]["count"]):
+    actor_number = actor_number + 1
+    helm_resource(
+        "gateway" + str(actor_number),
+        chart_prefix + "appgate-server",
+        flags=[
+            "--values=./localnet/kubernetes/values-common.yaml",
+            "--values=./localnet/kubernetes/values-gateway.yaml",
+            "--set=config.signing_key=gateway" + str(actor_number),
+        ],
+        image_deps=["poktrolld"],
+        image_keys=[("image.repository", "image.tag")],
+    )
+    k8s_resource(
+        "gateway" + str(actor_number),
+        labels=["gateways"],
+        resource_deps=["validator"],
+        port_forwards=[
+            str(42078+actor_number)+":42069", # gateway1 - exposes 42079, gateway2 exposes 42080, etc.
```

Comment:
Would `42069` collide with the appgateserver port or is this intentional?

Reply:
@Olshansk this is intentional. `42069` is the port we expose the service on inside of the container, which is static on all gateways and appgate-servers. The port that we forward to is dynamic, though!